usermods/audioreactive/audio_reactive.h

```diff
@@ -174,6 +174,13 @@ static float   fftResultMax[NUM_GEQ_CHANNELS] = {0.0f};               // A table
 #endif
 
 // audio source parameters and constant
+#ifdef CONFIG_IDF_TARGET_ESP32C3
+constexpr SRate_t SAMPLE_RATE = 16000;        // 16kHz - use if FFTtask takes more than 20ms. Physical sample time -> 32ms
+#define FFT_MIN_CYCLE 30                      // Use with 16Khz sampling
+#elif defined(CONFIG_IDF_TARGET_ESP32S2)
+constexpr SRate_t SAMPLE_RATE = 20480;        // Base sample rate in Hz - 20Khz is experimental.    Physical sample time -> 25ms
```

Comment:
@blazoncek I think this modification is not related to ESP-NOW.
Why would you use a non-standard sampling rate on -S2 ?

Reply:
FFT time on S2 is about 20ms.
I already have too many open branches and having one more was out of the question.

Reply:
You could use 16000 same as -C3. Or 18000 maybe.